lib/passport-wsfed-saml2/strategy.js

```diff
@@ -202,8 +202,7 @@ WsFedSaml2Strategy.prototype.authenticate = function (req, opts) {
   }
 
   function executeSamlp(req) {
-    if (req.body && ((req.method === 'POST' && req.body.SAMLResponse) ||
-                     (req.method === 'GET' && req.query.SAMLResponse))) {
+    if (req.body && req.method === 'POST' && req.body.SAMLResponse) {
```

Comment:
This was added into our fork for a specific reason. I don't think we can remove this.

Reply:
I've reverted for now, but I did repeat the debug runs to confirm that the req.method hasn't yet been a 'GET' in my testing.